ethereum/contracts/CappedMintableToken.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity >=0.4.22 <0.6.0;
```

Comment:
I would recommend that you use a specific versions i.e. `pragma solidity  ^0.5.7;`